src/index.html

```diff
@@ -15,17 +15,17 @@
       </div>
       <h1 class="header-title">What's Cookin'?</h1>
       <section class="user-info">
-       <h1 class='user-name'> Welcome</h1>
+       <p class='user-name'> Welcome</p>
       </section>
     </header>
     <section class="open-menu-selections x">
       <article class="menu-my-pantry menu-selection x">
         <h2 class="menu-my-pantry-title"><a href="./pantry.html">My Pantry</a></h2>
       </article>
-      <article class="menu-my-upcoming-recipes menu-selection hidden">
+      <article class="menu-my-upcoming-recipes menu-selection">
         <h3 class="menu-my-upcoming-recipes-title">My Upcoming Recipes</h2>
       </article>
-      <article class="menu-favorites menu-selection hidden">
+      <article class="menu-favorites menu-selection">
```

Comment:
Let's make sure we add that `.hidden` class back in so that we have what we need on initial page load.